Cargo.toml

```
@@ -196,161 +196,161 @@ rust-version = "1.83.0"
 
 [workspace.dependencies]
 aead = { version = "0.5", default-features = false }
-aes = "0.8.4"
-aes-gcm = "0.10.3"
-aligned-cmov = "2.3"
-anyhow = "1.0.80"
-assert_cmd = "2.0.14"
-assert_matches = "1.5.0"
+aes = "0.8"
+aes-gcm = "0.10"
+aligned-cmov = "2"
+anyhow = "1"
+assert_cmd = "2"
+assert_matches = "1"
 async-channel = "2"
 backtrace = "0.3"
-balanced-tree-index = "2.3"
+balanced-tree-index = "2"
 base64 = { version = "0.21", default-features = false }
-bincode = "1.3"
-bitflags = { version = "2.4", default-features = false }
-blake2 = { version = "0.10.6", default-features = false }
-bs58 = "0.4.0"
+bincode = "1"
+bitflags = { version = "2", default-features = false }
+blake2 = { version = "0.10", default-features = false }
+bs58 = "0.4"
 bulletproofs-og = { version = "3.0.0-pre.1", default-features = false }
-cargo-emit = "0.2.1"
+cargo-emit = "0.2"
 cargo_metadata = "0.18"
 cc = "1"
-cfg-if = "1.0"
-chrono = { version = "0.4.34", default-features = false }
-clap = "4.5.1"
-clio = "0.3.5"
+cfg-if = "1"
+chrono = { version = "0.4", default-features = false }
+clap = "4"
+clio = "0.3"
 cookie = "0.18"
-crc = { version = "3.0.1", default-features = false }
+crc = { version = "3", default-features = false }
 criterion = "0.5"
 crossbeam-channel = "0.5"
-ctr = "0.9.2"
-ctrlc = "3.4"
-curve25519-dalek = { version = "4.1.3", default-features = false }
-der = { version = "0.7.8", default-features = false }
-diesel = "2.1.4"
-diesel-derive-enum = "2.1.0"
+ctr = "0.9"
+ctrlc = "3"
+curve25519-dalek = { version = "4", default-features = false }
+der = { version = "0.7", default-features = false }
+diesel = "2"
+diesel-derive-enum = "2"
 diesel_migrations = "2.1.0"
```

not sure if this one was left on purpose
```suggestion
diesel_migrations = "2"
```

---

oops. no, thanks for catching that and the other one